docs/openapi/components/schemas/common/ChemicalComposition.yml

```
@@ -0,0 +1,153 @@
+$linkedData:
+  term: ChemicalComposition
+  '@id': https://w3id.org/traceability#ChemicalComposition
+title: Chemical Composition
+description: Measured chemical composition; all values in W%. 
```

Im guessing W% is a typo?

---

I think it stands for "[Weight percentage](https://www.google.com/search?q=%22W%25%22+weight)"... which would probably be better written out, here.
```suggestion
description: Measured chemical composition; all values in weight percentage (W%). 
```